CHANGELOG.md

```diff
@@ -2,12 +2,97 @@
+- Remove deprecated `rerun_demo` package [#4293](https://github.com/rerun-io/rerun/pull/4293)
+- Python: don't catch `KeyboardInterrupt` and `SystemExit` [#4333](https://github.com/rerun-io/rerun/pull/4333) (thanks [@Dvad](https://github.com/Dvad)!)
+
+#### 🪳 Bug Fixes
+- Fix line & points (& depth clouds points) radii being unaffected by scale & projection via Pinhole [#4199](https://github.com/rerun-io/rerun/pull/4199)
+- Fix inaccessible entities being incorrectly added to space view [#4226](https://github.com/rerun-io/rerun/pull/4226)
+- Silence spammy blueprint warnings and validate blueprint on load [#4303](https://github.com/rerun-io/rerun/pull/4303)
+- Fix markdown heading size [#4178](https://github.com/rerun-io/rerun/pull/4178)
+
+#### 🌁 Viewer Improvements
+- Add command to copy direct link to fully qualified URL [#4165](https://github.com/rerun-io/rerun/pull/4165)
+- Implement recording/last-modified-at aware garbage collection [#4183](https://github.com/rerun-io/rerun/pull/4183)
+
+#### 🖼 UI Improvements
+- Improve Visible History to support more general time queries [#4123](https://github.com/rerun-io/rerun/pull/4123)
+- Add support for Visible History to time series space views [#4179](https://github.com/rerun-io/rerun/pull/4179)
+- Make Visible History UI more ergonomic and show inherited values [#4222](https://github.com/rerun-io/rerun/pull/4222)
+- Display Visible History on timeline when the mouse hovers the UI [#4259](https://github.com/rerun-io/rerun/pull/4259)
+- Improve the Selection Panel with better title, context, and Space View key properties [#4324](https://github.com/rerun-io/rerun/pull/4324)
+
+#### 🕸️ Web
+- Put web viewer on `npm` [#4003](https://github.com/rerun-io/rerun/pull/4003)
+- Auto-switch port when getting AddrInUse error [#4314](https://github.com/rerun-io/rerun/pull/4314) (thanks [@dangush](https://github.com/dangush)!)
+- Generate per-PR web apps [#4341](https://github.com/rerun-io/rerun/pull/4341)
+
+#### 🧑‍💻 Dev-experience
+- Simple logging benchmarks for C++ & Rust [#4181](https://github.com/rerun-io/rerun/pull/4181)
+- New debug option to show the blueprint in the streams view [#4189](https://github.com/rerun-io/rerun/pull/4189)
+- Use pixi over setup scripts on CI + local dev [#4302](https://github.com/rerun-io/rerun/pull/4302)
+- Run deploy docs jobs serially [#4232](https://github.com/rerun-io/rerun/pull/4232)
+- fix windows test config on main [#4242](https://github.com/rerun-io/rerun/pull/4242)
+
+#### 🗣 Refactors
+- `StoreView` -> `StoreSubscriber` [#4234](https://github.com/rerun-io/rerun/pull/4234)
+- `DataStore` introduce `StoreEvent`s [#4203](https://github.com/rerun-io/rerun/pull/4203)
+- `DataStore` introduce `StoreView`s [#4205](https://github.com/rerun-io/rerun/pull/4205)
 
 ## [0.10.1](https://github.com/rerun-io/rerun/compare/0.10.0...0.10.1)
 
 ### Overview & Highlights
 This is a small release primarily to tie up some loose ends for our C++ SDK.
 
+### Details
+
```

Comment:
```suggestion
```

Reply:
you don't like the details headings? Ok 🤷 found it weird that logically this makes the details part of the highlights